crates/spin-primitives/README.md

```diff
@@ -0,0 +1,3 @@
+Primitives for SPIN.
+
+License: Apache-2.0
```

Comment:
Please remove license mentioning from here, we have the overall repo license.
https://github.com/QuantumFusion-network/qf-solochain/blob/main/LICENSE